cmake/CeleritasUtils.cmake

```diff
@@ -357,6 +372,11 @@ function(celeritas_define_options var doc)
       )
     endif()
   endif()
+  set(_last_var _LAST_${var})
+  if(NOT ${var} STREQUAL ${_last_var})
```

Comment:
```suggestion
  if(NOT var STREQUAL _last_var)
```

Reply:
This is actually correct as written: we want to compare `"${{var}}"` to `"${${_last_var}}"`, but I guess I could use the phrasing I just gave to improve clarity?

Reply:
I see.

> I could use the phrasing

Either that of a comment calling out the oddness (but correctness) of the syntax.